code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -467,6 +467,18 @@
 			if ("listtarget")
 				src.show_text("salute, bow, hug, wave, glare, stare, look, leer, nod, tweak, flipoff, doubleflip, shakefist, handshake, daps, slap, boggle, highfive")
 
+			if ("lizard")
```

Comment:
This could instead be one statement that checks the player's mutantrace, and returns either the correct text for the mutantrace, or some kind of admonishment to the player if they aren't a mutantrace. And maybe some kind of flavortext for mutantraces without unique emotes.